src/sentry/runner/commands/configoptions.py

```diff
@@ -0,0 +1,271 @@
+import re
+
+import click
+
+from sentry.runner.decorators import configuration
+
+tracked = [
+    re.compile("sentry:test_key.+"),
```

Comment:
Regex is cool, but I think our lives would be a lot easier if we just hardcode the options we want to ignore here. There's also flags we can check `not ((opt.flags & manager.FLAG_NOSTORE) and (opt.flags & manager.FLAG_IMMUTABLE))` but I understand if our faith in that isn't that strong.

Reply:
Assuming you are using this to identify tracked options, I don't think either a regex or a list here would be a good idea. IF we need to add a filed to the options data structure so be it. No issue with that. It would be a lot more robust and harder to mess up than a regex that is defined far away from where the options are registered.

Regardless, I thought we agreed in the meeting that we would achieve this by recording who changed an option and prevent the deployment from going ahead if we tried to modify an option that was changed by anything other than the tool. Which means we should not need a list of excluded options. Am I misremembering ?